Add missing build dependency to make sure CommitWindow.h is found

Fixes a build failure:

| Applications/commit/src/commit.mm:1:10: fatal error: 'CommitWindow/CommitWindow.h' file not found
| #include <CommitWindow/CommitWindow.h>
|          ^
| 1 error generated.

Upstream-Status: Pending
--- Applications/commit/target.orig	2016-08-06 20:55:29.000000000 +0200
+++ Applications/commit/target	2016-08-06 20:55:58.000000000 +0200
@@ -1 +1,2 @@
 SOURCES = src/*.mm
+LINK   += CommitWindow
